lib/src/execute.rs

```diff
@@ -49,7 +49,7 @@ pub const EPOCH_INTERRUPTION_PERIOD: Duration = Duration::from_micros(50);
 
 enum Instance {
     Module(Module, InstancePre<WasmCtx>),
-    Component(component::InstancePre<ComponentCtx>),
+    Component(compute::ComputePre<ComponentCtx>),
```

Comment:
I think this is the most notable change, as we get the opportunity to amortize the component export lookups on the insance pre now.